pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -67,7 +67,6 @@ let
         --output_user_root="$bazelUserRoot" \
         ${cmd} \
         --curses=no \
-        -j $NIX_BUILD_CORES \
```

Comment:
https://github.com/NixOS/nixpkgs/pull/227432/checks?check_run_id=13035938483
aarch64-linux build of bazel-watcher worked with old code but x86_64 was failing without this change :thinking: https://github.com/NixOS/nixpkgs/pull/227432/checks?check_run_id=13035942786
```
       > no configure script, doing nothing
       > building
       > patching script interpreter paths in .
       > ./release/release.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/zlf0f88vj30sc7567b80l52d19pbdmy2-bash-5.2-p15/bin/bash"
       > ./release/npm/package.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/zlf0f88vj30sc7567b80l52d19pbdmy2-bash-5.2-p15/bin/bash"
       > ./tools/workplace_status.py: interpreter directive changed from "#! /usr/bin/env python3" to "/nix/store/fdqpyj613dr0v1l1lrzqhzay7sk4xg87-python3-3.10.10/bin/python3"
       > Extracting Bazel installation...
       > ERROR: -j :: Unrecognized option: -j
       For full logs, run 'nix log /nix/store/8db0365qwz8j1fdrp5d0fb5hajjhxp5j-bazel-watcher-0.21.2-deps.tar.gz.drv'.
error: 1 dependencies of derivation '/nix/store/grg2xai3g2gzkk7vr0crgqagsqjm9ljl-bazel-watcher-0.21.2.drv' failed to build
```